e2e/tests/general/network-switch.ts

```diff
@@ -0,0 +1,142 @@
+import * as utils from '../../utils/utils';
+
+/**
+ * Network Switch
+ */
+export const networkSwitch = async () => {
```

Comment:
it's really hard to follow this...
The tests should be like this:
```ts
describe('Network switch', () => {
  ...
  test('it works ok on ethereum', () => {
    // call util function to do something reusable
    // check everything is fine

    // call util function to do something reusable
    // check everything is fine
  }

  test('it works on zilliqa', () => {
    // call util function to do something reusable
    // check everything is fine
  }
}
```

Try to use this pattern, it would be more ease to follow in case of a failing test

Reply:
You have to separate the test logic and util function that you use in multiple tests.